src/reducers/dataEntryErrorsReducer.js

```diff
@@ -3,6 +3,16 @@ import {
   UPDATE_SERVICE_ERROR,
   DISMISS_DATA_ENTRY_ERRORS,
 } from '../actions';
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({
```

Comment:
Could be nice to initialize this in its own file under `services`, like we do for e.g. Mixpanel (in `services/analytics.js`), so a specific reducer doesn't have to own the concern of configuring Sentry.

Reply:
Fixed in 5559ebbb6adaaaa09ee40e7a70e02f1cc66b06f6